### ISSUE TITLE
`erase` and `noErase` function implementation

### ISSUE THREAD

**Issue by quinton-ashley** [q5js/q5.js#33]

**Comment by quinton-ashley** [comment]
@Tezumie I'm a bit confused about what this code is doing:

```js
if ($._fillSet) {
	$.ctx.fillStyle = $.color($.ctx.fillStyle).toString();
}
if ($._strokeSet) {
	$.ctx.strokeStyle = $.color($.ctx.strokeStyle).toString();
}
```

It seems like it would do nothing in cases where the fill and stroke styles can be parse by color. Do you mean to put back the original fill and stroke that the user had set before using `erase` ?

**Comment by Tezumie** [comment]
Good catch! its not necessary.

**Comment by quinton-ashley** [comment]
@Tezumie p5's `noErase` actually resets the original fill and stroke

https://p5js.org/reference/#/p5/noErase

**Comment by quinton-ashley** [comment]
fixed in v2.0-beta11
